README.md

```diff
@@ -136,40 +136,48 @@ directly with `poetry` rather than go through the package.
 brew install mysql postgresql # MacOS dependencies for C bindings
 poetry install
 ```
-
-**2. Download CSV of Testing Data**
+**2. Start Databases**
 
 ```shell-session
-wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
-unzip ml-25m.zip -d dev/
+docker-compose up -d mysql postgres
 ```
 
-**3. Start Databases**
+**3. Run Unit Tests**
 
 ```shell-session
-docker-compose up -d mysql postgres
+poetry run python3 -m unittest
 ```
 
-**4. Run Unit Tests**
+**4. Seed the Database(s)**
+
+First, download the CSVs of seeding data:
 
 ```shell-session
-poetry run python3 -m unittest
-```
+wget https://files.grouplens.org/datasets/movielens/ml-25m.zip
+unzip ml-25m.zip -d dev/
+
+# By default, we insert 25M rows which takes manu minutes
+# Optional, but recommended, only 1M rows for speed
+# Seedings always happens from `ratings.csv`, get 1M rows (+header)
+mv dev/ml-25m/ratings.csv dev/ml-25m/ratings_25m.csv
+head -n 1000001 dev/ml-25m/ratings_25m.csv > dev/ml-25m/ratings.csv
 
-**5. Seed the Database(s)**
+```
 
-If you're just testing, we recommend just setting up one database (e.g.
-Postgres) to avoid incurring the long setup time repeatedly.
+Now you can insert it into the testing database(s):
 
 ```shell-session
 preql -f dev/prepare_db.pql postgres://postgres:Password1@127.0.0.1:5432/postgres
+# You can compare in just one if you like, you don't need to seed them all.
 preql -f dev/prepare_db.pql mysql://mysql:Password1@127.0.0.1:3306/mysql
+
+# These typically would run remotely
 preql -f dev/prepare_db.psq snowflake://<uri>
 preql -f dev/prepare_db.psq mssql://<uri>
 preql -f dev/prepare_db_bigquery.pql bigquery:///<project> # Bigquery has its own
 ```
 
-**6. Run data-diff against seeded database**
+**5. Run `data-diff` against seeded database**
```

Comment:
I actually think it's better to leave it as text, rather than code, as it's a name. Maybe put it in bold instead?